plumber-android/src/main/java/leakcanary/AndroidLeakFixes.kt

```diff
@@ -198,12 +205,223 @@ enum class AndroidLeakFixes {
     }
   },
 
+  /**
+   * ActivityChooserModel holds a static reference to the last set ActivityChooserModelPolicy
+   * which can be an activity context.
+   * Tracked here: https://code.google.com/p/android/issues/detail?id=172659
+   */
+  ACTIVITY_CHOOSER {
```

Comment:
Comment should ideally describe what the fix is.

Here it looks like: every time an activity is destroyed, we get the activity chooser model for ShareActionProvider.DEFAULT_SHARE_HISTORY_FILE_NAME and set its OnChooseActivityListener to null?

Can you link to the code that's setting up this listener in the first place, creating the leak?

Reply:
As for a good part of this fixes, this was taken from the hack suggested in `AndroidReferenceMatchers`. However, after reviewing the source a little, I was unable to determine properly how this leak happens in the wild. I'll delete this part until I can properly determine if it's needed or not